content/rest/usage.textile

```diff
@@ -316,6 +319,10 @@ blang[java,ruby,php,csharp].
 
   Failure to retrieve the Ably server time will raise an "@AblyException@":/rest/types#ably-exception.
 
+<div lang="jsall">
+<%= partial 'rest/_request' %>
+</div>
+
 h2(#related-types). Related types
```

Comment:
You have not included the `HttpPaginatedResponse` type in Related types.  Also, you have not added this method to the Realtime docs.  This method is available for both REST & Realtime